unicodetools/src/main/java/org/unicode/tools/emoji/CountEmoji.java

```diff
@@ -382,17 +382,23 @@ enum Attribute {
         zwj("Ⓩ"),
         gender(Emoji.FEMALE),
         role(Emoji.WOMAN_STR),
-        family(Emoji.NEUTRAL_FAMILY),
+        family(Emoji.NEUTRAL_FAMILY, "group"),
```

Comment:
This is a minor fix to chart tooltips, which currently call any multi-person group a family.